docs/src/Toast.doc.js

```diff
@@ -46,7 +46,7 @@ card(
         name: 'text',
         type: 'string | Array<string>',
```

Comment:
This is incorrect, the type is   `text: string | Element<*>,`

Reply:
I was wondering about that - will update : )